roles/tackle/tasks/main.yml

```diff
@@ -190,10 +188,6 @@
 
     - when: (rhsso_keycloak_client_status.resources | length) == 0
       block:
-        - name: "Generate random values for RHSSO Hub Client secret"
-          set_fact:
-            keycloak_sso_hub_client_secret: "{{ lookup('password', '/dev/null chars=ascii_lowercase,ascii_uppercase,digits length=32') }}"
-
```

Comment:
I assume we are not expecting (at all) to have a secret set for the only remaining keycloak UI client correct?

Reply:
Right, the UI client doesn't have a secret.